File: 1st semester/PF lab/lab 8/Q3.cpp
#include<stdio.h>
int main()
{
	int size,i,j,sum,avg;
	printf("enter the size of array: ");
	scanf("%d",&size);
	int a[size];
	for (i=0;i<size;i++)
	{
		printf("Enter (%d) number in array:",i);
		scanf("%d",&a[i]);
	}
	printf("elements in array are: ");
	for (j=0;j<size;j++)
	{
	printf("%d ,",a[j]);
	sum+=a[j];
	avg=sum/size;
	}
	printf("\nsum of all elements: %d\n",sum);
	printf("Average: %d\n",avg);
}

File: 2 semester/OOP lab/Lab 1/q3.java
import java.util.Scanner;

public class q3 {
    public static void main(String[] args) {
        System.out.println("Enter variable x");
        int x;
        Scanner A = new Scanner(System.in);
        x = A.nextInt();
        System.out.println("Enter variable y");
        int y;
        y = A.nextInt();
        System.out.println("Enter variable z");
        int z;
        z = A.nextInt();
        int sum, product;
        double avg;

        sum = x+y+z;
        product = x*y*z;
        avg = (x+y+z) / 3 ;

        System.out.println("sum = " +sum);
        System.out.println("Product = "+product);
        System.out.println("Average = "+avg);

    }
}